pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -42,28 +41,128 @@ type VolumeZone struct {
 }
 
 var _ framework.FilterPlugin = &VolumeZone{}
+var _ framework.PreFilterPlugin = &VolumeZone{}
 var _ framework.EnqueueExtensions = &VolumeZone{}
 
 const (
 	// Name is the name of the plugin used in the plugin registry and configurations.
 	Name = names.VolumeZone
 
+	stateKey framework.StateKey = Name
+
 	// ErrReasonConflict is used for NoVolumeZoneConflict predicate error.
 	ErrReasonConflict = "node(s) had no available volume zone"
 )
 
-var volumeZoneLabels = sets.NewString(
+// pvTopology holds the value of a pv's topologyLabel
+type pvTopology struct {
+	pvName     string
+	labelKey   string
+	labelValue string
```

Comment:
```suggestion
	key    string
	values []string
```

Reply:
Excuse me if this is a stupid question :(
I think the key and the value is a one-to-one correspondence `Labels map[string]string`
Why are we using `[]string`, are you suggesting convert `volumeVSet` to a slice for every label?

Reply:
yes, either as a `[]string` or `sets.String`. Whatever works better.

Reply:
I'm going with `sets.String` as I'm not sure how large `volumeVSet` can be.